examples/community/README.md

```diff
@@ -5,4 +5,6 @@
 | Example   |      Description      |      Author      |    Colab  |
 |:----------|:----------------------|:-----------------|----------:|
 | CLIP Guided Stable Diffusion | Doing CLIP guidance for text to image generation with Stable Diffusion| [Suraj Patil](https://github.com/patil-suraj/) | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/huggingface/notebooks/blob/main/diffusers/CLIP_Guided_Stable_diffusion_with_diffusers.ipynb) |
-| One Step U-Net (Dummy) | [Patrick von Platen](https://github.com/patrickvonplaten/) | - |
+| One Step U-Net (Dummy) | - | [Patrick von Platen](https://github.com/patrickvonplaten/) | - |
```

Comment:
btw fixed this table here... @patrickvonplaten your name was under the `Description` column.